src/http/index.js

```diff
@@ -142,7 +141,7 @@ function serializeHeaderValue(value) {
 //     Cookie: two
 //  =  { Cookie: [ "one", "two" ]
 export function serializeHeaders(headers = {}) {
-  if (!isFunction(headers.entries)) return {};
+  if (typeof headers.entries !== 'function') return {};
```

Comment:
I had a brief note on the predicates like this in my original comment: https://github.com/swagger-api/swagger-js/issues/2187

Given that we have `valid-typeof` eslint rule active, the correctness of these assertions is guaranteed; so let's go for it.